packages/utils/test/lighthouserc.test.js

```diff
@@ -13,23 +13,30 @@ const rc = require('../src/lighthouserc.js');
 const {safeDeleteFile} = require('../../cli/test/test-utils.js');
 
 describe('lighthouserc.js', () => {
+  function writeFile(path, content) {
+    fs.writeFileSync(path, content);
+  }
   describe('#loadAndParseRcFile', () => {
-    let tmpFile;
-    function writeFile(json) {
-      fs.writeFileSync(tmpFile, JSON.stringify(json));
+    function writeJSONFile(path, json) {
```

Comment:
let's use a `writeYAMLFile` helper too that uses `yaml.safeDump` instead of `JSON.stringify` so we can share the test definitions between them :)

Reply:
Done.